Remove ipa_host variable from manage-participants

makirill opened the pull request:
The ipa_host variable should be set at main entry point based on
ocp_subdomain and subdomain_base_suffix variables provided externally.

makirill committed:
Remove ipa_host variable from manage-participants

The ipa_host variable should be set at main entry point based on
ocp_subdomain and subdomain_base_suffix variables provided externally.

makirill committed:
Adding checks for ocp_subdomain and subdomain_base_suffix vars

oybed commented on manage-participants/main.yml:
```diff
@@ -3,6 +3,14 @@
+      fail:
+        msg: "ocp_subdomain and subdomain_base_suffix vars needs to be provided in order to continue withe the playbook"
+      when:
+        - ocp_subdomain is undefined or subdomain_base_suffix is undefined
```
let's also check for "empty" variables here - i.e.: `ocp_subdomain|trim == ""` 

Also would be good to include the output of the variables in the message to know which one is empty - i.e.: something like the following:
```
msg: "Variable foo ({{ foo|default('')}}) or bar ({{ bar|default('')}}) is not defined or empty"
```

makirill replied:
Fixed in the 458b368afac80eb7e2806fd17cbbe2ecf34eb0f8 commit.

makirill committed:
Adding additional information to the variables check

- Adding empty check for ocp_subdomain and subdomain_base_suffix
  variables
- Additional output for error message

oybed committed:
Minor formatting update

oybed reviewed (approved):
LGTM
